src/core/instance/lifecycle.js

```diff
@@ -105,7 +105,7 @@ export function lifecycleMixin (Vue) {
   }
 
   Vue.prototype.$forceUpdate = function () {
-    this._watcher.update()
+    this._update(this._render())
```

Comment:
Directly call `_update` with `_render` result. In some scenarios `_watcher` was not yet defined. This is also faster anyway.